stdlib/stdlib.ispc

```diff
@@ -1658,174 +1658,224 @@ __declspec(safe) static inline uniform float16 rcp(uniform float16 v) {
 }
 
 __declspec(safe, cost2) static inline float16 fmod(float16 x, float16 y) {
-    // Handle special cases: NaN and division by zero
-    if (isnan(x) || isnan(y) || y == 0.0f) {
-        return (signbits(x) == 0) ? float16bits((uniform int16)0x7E00) : float16bits((uniform int16)0xFE00);
+    // Handle special cases: NaN inputs or y == 0.0f16
+    if (isnan(x) || isnan(y) || y == 0.0f16) {
```

Comment:
Are we missing the case with `y == -0.0f16` here? I forgot how comparison with 0 works according to IEEE754 standard - if == matching for both zeros or not. Could you please double check?

Reply:
According to standard positive zero (+0.0) and negative zero (-0.0) are considered equal when compared using `==`. I also think that in `if (x == 0.0f16 || x == -0.0f16)` below it's sufficient to have only `x == 0.0f16`.

Reply:
Thanks for double-checking.